supabase/tests/67_test_public_channel_guard.sql

```
@@ -278,31 +278,42 @@ SELECT throws_ok(
-  'channel-admin can update non-public settings on a private channel'
+-- 4) Preview key cannot mutate channel settings without channel.update_settings.
+SELECT tests.clear_authentication();
+SELECT set_config('request.jwt.claim.role', 'anon', true);
+SELECT set_config('request.headers', '{"capgkey":"public-channel-guard-preview-key"}', true);
+
+UPDATE public.channels
+SET allow_emulator = true
+WHERE id = 6700401;
+
+SELECT is(
+  (SELECT allow_emulator FROM public.channels WHERE id = 6700401),
```

SQLFluff (4.2.2)

[error] The 'WHERE' keyword should always start a new line.

(LT14)